pkg/services/live/live.go

```diff
@@ -326,6 +327,10 @@ func (g *GrafanaLive) handleOnSubscribe(client *centrifuge.Client, e centrifuge.
 
 	handler, addr, err := g.GetChannelHandler(user, channel)
 	if err != nil {
+		if errors.Is(err, live.ErrInvalidChannelID) {
+			logger.Info("Invalid channel ID", "user", client.UserID(), "client", client.ID(), "channel", e.Channel)
```

Comment:
Nit. Error or debug level?

Reply:
This should not be an error since it's a validation, but we also need administrators to see that such errors happen - that's why I have chosen INFO supposing that DEBUG is off in many setups

Reply:
I think we treat all other validation errors as error level in logs. Info tends to be too spammy. But yeah no biggie